bayes/src/main/java/com/enigmastation/ml/bayes/FisherClassifier.java

```diff
@@ -15,6 +15,9 @@
  */
 package com.enigmastation.ml.bayes;
 
+/**
```

Comment:
Agreed. Needs a pointer to what the Fisher modification to the Bayes algorithm is.